gpii/node_modules/settingsHandlers/test/WebSocketsSettingsHandlerTests.js

```diff
@@ -0,0 +1,125 @@
+/*
+ * WebSockets Settings Handler Tests
+ *
+ * Copyright 2013, 2014 Emergya
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * The research leading to these results has received funding from the European Union's
+ * Seventh Framework Programme (FP7/2007-2013)
+ * under grant agreement no. 289016.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/GPII/universal/blob/master/LICENSE.txt
+ */
+
+"use strict";
+
+var fluid = require("universal"),
+    gpii = fluid.registerNamespace("gpii"),
+    jqUnit = fluid.require("jqUnit");
+
+fluid.registerNamespace("gpii.settingsHandlers");
+
+var wsSettingsHandler = fluid.registerNamespace("gpii.settingsHandlers.WebSocketsSettingsHandler");
+
+jqUnit.module("WebSockets settings handler test module");
+
+jqUnit.test("Checking wsSettingsHandler object before doing anything", function () {
+    jqUnit.assertTrue("There are an object to store the settings", wsSettingsHandler.settings);
+    jqUnit.assertTrue("There are an object to store the clients", wsSettingsHandler.clients);
+
+    jqUnit.assertDeepEq("wsSettingsHandler.settings is empty", {}, wsSettingsHandler.settings);
+    jqUnit.assertDeepEq("wsSettingsHandler.clients is empty", {}, wsSettingsHandler.clients);
+});
+
+jqUnit.test("Checking wsSettingsHandler's get/set methods", function () {
+    var getPayload = [
```

Comment:
These payloads should go into our gpii.tests namespace above rather than appearing as giant local variables